modules/installation-osp-creating-network-resources.adoc

```diff
@@ -10,80 +10,24 @@ Create the network resources that an {product-title} on {rh-openstack-first} ins
 
 .Prerequisites
 
-* Python 3 is installed on your machine.
 * You downloaded the modules in "Downloading playbook dependencies".
 * You downloaded the playbooks in "Downloading the installation playbooks".
 
 .Procedure
 
-. Optional: Add an external network value to the `inventory.yaml` playbook:
+* On a command line, create the network resources by running the following command:
 +
-.Example external network value in the `inventory.yaml` Ansible playbook
-[source,yaml]
+[source,terminal]
 ----
-...
-      # The public network providing connectivity to the cluster. If not
-      # provided, the cluster external connectivity must be provided in another
-      # way.
-
-      # Required for os_api_fip, os_ingress_fip, os_bootstrap_fip.
-      os_external_network: 'external'
-...
+$ ansible-playbook -i inventory.yaml network.yaml
 ----
 +
-[IMPORTANT]
+[NOTE]
 ====
-If you did not provide a value for `os_external_network` in the `inventory.yaml` file, you must ensure that VMs can access Glance and an external connection yourself.
+The API and Ingress VIP fields will be overwritten in the `inventory.yaml` playbook with the IP addresses assigned to the network ports.
 ====
-
-. Optional: Add external network and floating IP (FIP) address values to the `inventory.yaml` playbook:
-+
-.Example FIP values in the `inventory.yaml` Ansible playbook
-[source,yaml]
-----
-...
-      # OpenShift API floating IP address. If this value is non-empty, the
-      # corresponding floating IP will be attached to the Control Plane to
-      # serve the OpenShift API.
-      os_api_fip: '203.0.113.23'
-
-      # OpenShift Ingress floating IP address. If this value is non-empty, the
-      # corresponding floating IP will be attached to the worker nodes to serve
-      # the applications.
-      os_ingress_fip: '203.0.113.19'
-
-      # If this value is non-empty, the corresponding floating IP will be
-      # attached to the bootstrap machine. This is needed for collecting logs
-      # in case of install failure.
-      os_bootstrap_fip: '203.0.113.20'
-----
 +
-[IMPORTANT]
+[NOTE]
 ====
-If you do not define values for `os_api_fip` and `os_ingress_fip`, you must perform postinstallation network configuration.
-
-If you do not define a value for `os_bootstrap_fip`, the installer cannot download debugging information from failed installations.
-
-See "Enabling access to the environment" for more information.
+The `inventory.yaml` and `network.yaml` playbooks will be deleted by the `down-network.yaml`.
```

Comment:
Perhaps it would be informationa to mention here, that before running `network.yaml` playbook, user can configure additional subnet for dualstack by uncommenting and configuring `os_subnet6_range` and optionally tweak configure options:

- `os_subnet6_address_mode`
- `os_subnet6_router_advertisements_mode`
- `service_subnet6_range`
- `cluster_network6_cidr`
- `cluster_network6_prefix`

in `inventory.yaml`.

Reply:
I added it to the latest updates. [Creating network resources on RHOSP](https://file.rdu.redhat.com/antaylor/OSDOCS-9146/installing/installing_openstack/installing-openstack-user.html#installation-osp-creating-network-resources_installing-openstack-user)
![image](https://github.com/openshift/openshift-docs/assets/46728411/600301fb-d207-4c51-b134-ce3094c8bf38)